docs/sources/getting-started/build-influxdb-dashboard.md

```diff
@@ -1,30 +1,29 @@
 +++
 aliases = ["/docs/grafana/latest/getting-started/getting-started-influxdb/"]
-description = "Guide for getting started with Grafana and InfluxDB"
-keywords = ["grafana", "intro", "guide", "started"]
-title = "With Grafana and InfluxDB"
+description = "Learn how to build your first InfluxDB dashboard in Grafana."
+title = "Build an InfluxDB dashboard"
 weight = 250
 +++
 
-# Getting started with Grafana and InfluxDB
+# Build an InfluxDB dashboard
```

Comment:
I think the original titles where good these are much more than just building a influxdb dashboard